db/Seed/workoutSeed.json

```diff
@@ -0,0 +1,802 @@
+[
```

Comment:
Ah, fair point. I wonder if I can override the built-in hook and fake the 'created_at'. Will try.